examples/lock_exchange.jl

```diff
@@ -175,7 +175,7 @@ n = Observable(1)
 
 title = @lift @sprintf("t = %5.2f hours", times[$n] / hour)
 
-wₙ = @lift wt[$n]
+uₙ = @lift ut[$n]
```

Comment:
```suggestion
un = @lift ut[$n]
```

I don't like the look of the _n subscripts here but up to you if you want to keep it.